projects/ngrx.io/content/guide/signals/signal-store/index.md

```diff
@@ -312,8 +312,10 @@ export const BooksStore = signalStore(
           return booksService.getByQuery(query).pipe(
             tapResponse({
               next: (books) => patchState(store, { books }),
```

Comment:
We also need to set the loading state to `false` when the response is received.

```suggestion
              next: (books) => patchState(store, { books, isLoading: false }),
```

Reply:
thanks tim!